### PR TITLE
Adds `get-hotkey` sexp

### PR DESCRIPTION
**Pull request by wookieejedi**
This PR adds a sexp that returns the integer value of a ship or wing hotkey. No hotkey returns -1, F5 returns 0, F6 returns 1, ... The sexp takes one argument, the ship or wing name.

### CODE REVIEW

File: code/parse/sexp.cpp [hunk 1]
```diff
@@ -33765,6 +33797,14 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\t2:\t(Optional) Ignore the directive count set by any earlier SEXPs in the event. If set to false it will add instead\r\n"
 	},
 
+	// wookieejedi
+	{ OP_GET_HOTKEY, "get-hotkey\r\n"
+		"\tReturns the hotkey integer of the ship or wing.\r\n"
+		"\tNo hotkey returns -1, F5 returns 0, F6 returns 1, ...\r\n\r\n"
```
Comment (Goober5000) [inline code/parse/sexp.cpp]: You should indicate what the maximum possible returned value is, and what it corresponds to.

File: code/parse/sexp.cpp [hunk 2]
```diff
@@ -19805,6 +19806,28 @@ void multi_sexp_set_respawns()
...
+ * get a hotkey for a one ship or wing
+ */
+int sexp_get_hotkey(int node)
+{
+	object_ship_wing_point_team oswpt;
+	sexp_get_object_ship_wing_point_team(&oswpt, CTEXT(node));
+	int hotkey;
+
+	// if ship, just grab the hotkey number
+	// if wing, get the first ship in the wing, then grab hotkey
```
Comment (Goober5000) [inline code/parse/sexp.cpp]: This comment does not accurately describe what the code does.

File: code/parse/sexp.cpp [hunk 3]
```diff
@@ -33799,8 +33799,8 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 
 	// wookieejedi
 	{ OP_GET_HOTKEY, "get-hotkey\r\n"
-		"\tReturns the hotkey integer of the ship or wing.\r\n"
-		"\tNo hotkey returns -1, F5 returns 0, F6 returns 1, ...\r\n\r\n"
+		"\tReturns the hotkey integer of the ship or wing. Return values range from -1 to 8.\r\n"
+		"\tNo hotkey returns -1, F5 returns 0, F6 returns 1, ..., F12 returns 7, and 'Hidden' hotkey returns 8.\r\n\r\n"
```
Comment (Goober5000) [inline code/parse/sexp.cpp]: I don't think the 'Hidden' hotkey is actually a value that can be returned in-mission.  It's only used during mission load.

**Comment by Goober5000** [comment]
@wookieejedi is the "merge pull request" button visible to you?

**Comment by wookieejedi** [comment]
@Goober5000 Unfortunately, no.

**Comment by Goober5000** [comment]
Ah gotcha.  I'll merge it then.

**Comment by wookieejedi** [comment]
Thanks!

**Pull request merged**